### PR TITLE
Update Stats.jsx

### PR DESCRIPTION
**Pull request by deveshgoyal1000**
Changed numbers to:
$20,000+ prizes
30,000+ participants
30+ countries
200,000+ empowered

## Check List (Check all the boxes which are applicable) 
- [X] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: src/components/PageComp/EventsPage/Stats.jsx [hunk 1]
```diff
@@ -28,21 +28,22 @@ const Stats = () => {
           />
           <CounterCard
             counterOn={counterOn}
-            icon={FaFontAwesomeFlag}
-            number={70}
-            title='Countries'
+            icon={bsfilltrophyfill}
+            number={30,000}
+            title='Participants'
+
```
Comment (Chandraprakash-Darji) [inline src/components/PageComp/EventsPage/Stats.jsx:34]: remove extra line

File: src/components/PageComp/EventsPage/Stats.jsx [hunk 2]
```diff
@@ -28,21 +28,22 @@ const Stats = () => {
           />
           <CounterCard
             counterOn={counterOn}
-            icon={FaFontAwesomeFlag}
-            number={70}
-            title='Countries'
+            icon={bsfilltrophyfill}
+            number={30,000}
```
Comment (Chandraprakash-Darji) [inline src/components/PageComp/EventsPage/Stats.jsx:32]: remove `,` from number

File: src/components/PageComp/EventsPage/Stats.jsx [hunk 3]
```diff
@@ -28,21 +28,22 @@ const Stats = () => {
           />
           <CounterCard
             counterOn={counterOn}
-            icon={FaFontAwesomeFlag}
-            number={70}
-            title='Countries'
+            icon={bsfilltrophyfill}
```
Comment (Chandraprakash-Darji) [inline src/components/PageComp/EventsPage/Stats.jsx:31]: This icon is not imported

File: src/components/PageComp/EventsPage/Stats.jsx [hunk 4]
```diff
@@ -28,21 +28,22 @@ const Stats = () => {
...
+            icon={FaFontAwesomeFlag}
+            number={30}
+            title='Countries'
           />
           <CounterCard
             counterOn={counterOn}
-            icon={VscSymbolEvent}
-            number={10}
-            title='Events'
+            icon={IoIosPeople}
```
Comment (Chandraprakash-Darji) [inline src/components/PageComp/EventsPage/Stats.jsx:44]: This icon import is wrong

**Review by Chandraprakash-Darji — CHANGES_REQUESTED** [review]

**Comment by AnuragThePathak** [comment]
@deveshgoyal1000 are you still working on it?

**Comment by deveshgoyal1000** [comment]
> @deveshgoyal1000 are you still working on it?

nope can i close this

**Comment by deveshgoyal1000** [comment]
@AnuragThePathak 
nope can I close this

**Comment by AnuragThePathak** [comment]
No problem, I can do it for you.

**Pull request closed without merging**